fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -6215,6 +6216,26 @@ public void setHasForbitGlobalDict(String dbName, String tableName, boolean isFo
         }
     }
 
+    public void replayModifyTableColumn(short opCode, ModifyTableColumnOperationLog info) {
```

Comment:
why there are two method? `replayModifyHiveTableColumn` and `replayModifyTableColumn`